Torrent Data Refresh Button and Category View

AzukaChan opened the pull request:
Finally added a working refresh seed/peer data button, but gonna need to fix the branding as well as re-add "books" to the category list on en/view/torrent.php before merging. If you could suggest edit those changes when you get the chance @KevinJDurant that would be cool.

Just need to add a category view and things are starting to look pretty good.

AzukaChan committed:
Feature Update

Finally added a working refresh seed/peer data button, but gonna need to fix the branding on en/view/torrent.php

Milen4o commented:
I think to start workin on the template on localhost soon to give it more nice view but yeah the basic is there and working :) I was searchin for a monts how to create a torrent website and there's nothin on the internet. Later wil add backend admin template with all admin functions like new users, torrents, etc.

Milen4o commented:
Where you hosting your website ? How you deal with the law issues ? Here tis is veri tight.

AzukaChan committed:
Seed/Peer Refresh and Link fixes

Keeping everything 'OpenTorrentSite' but changed 'Copyright &copy; Your Website 2017' to github link and live site.

AzukaChan committed:
Category View

Users can now browse torrent categories by clicking on the category name from the index page. Currently setup to display 10 torrents per page. Previous / Next page buttons will appear once a category has 10 torrents, this can be changed from /en/view/category.php

AzukaChan commented:
@KevinJDurant Sorry my pulls are always so messy lol. I use live site to test and then reupload the entire file. Fixed most of the neutral branding but kept the github/live links down the bottom in hopes of other people finding this project. 

Added a category view that displays X amount of torrents per page which users can access by clicking on the category name from the index page. Previous / Next buttons will appear once a category reaches X amount of torrents. The torrents per page can be changed from en/view/category.php

Re-added books to search so you don't have to worry about that going missing.

AzukaChan committed:
Update torrent.php

Dropdown-menu script was missing causing a 404 when changing category in the search bar.
Re-added iTorrents link (was removed for live site) and fixed formatting

AzukaChan committed:
Update category.php

Dropdown-menu script was missing causing a 404 when changing category in the search bar.

AzukaChan committed:
Update index.php

Fixed branding and re-added itorrent links. 
'Copyright &copy; Your Website' has been replaced with Github / Live links.

See https://torrents.azukachan.com for live demo.
Note: itorrent links have been removed for the live demo.
